doc/en/components/charts/chart-features-navigation.md

```diff
@@ -0,0 +1,216 @@
+---
+title: $PlatformShort$ Data Chart | Data Visualization Tools | Navigation | Infragistics
+_description: Navigate Infragistics' $PlatformShort$ charts by panning right and left and zooming horizontally and vertically using mouse or touch. Learn about $ProductName$ graph navigation capabilities!
+_keywords: $PlatformShort$ charts, data chart, navigation, $ProductName$, Infragistics
+mentionedTypes: ['XamDataChart', 'ModifierKeys']
+---
+# $PlatformShort$ Chart Navigation
+
+In the $PlatformShort$ data chart component, navigation is disabled by default. Once enabled, you are able to zoom and pan the data chart either programmatically or in the UI.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/data-chart-chart-navigation" 
+           alt="$PlatformShort$ Navigation Example" 
+           github-src="charts/data-chart/chart-navigation">
+</code-view>
+
+<div class="divider--half"></div>
+
+In order to use navigation in the data chart, you need to import and register the interactivity module. This can be done by using the following code:
+
+```razor
+DataChartInteractivityModule.Register(IgniteUIBlazor);
+```
+
+```ts
+import { IgxDataChartInteractivityModule } from 'igniteui-angular-charts';
+
+@NgModule({
+    imports: [ IgxDataChartInteractivityModule ]
+})
+```
+
+```ts
+import { IgrDataChartInteractivityModule } from 'igniteui-react-charts';
+
+IgrDataChartInteractivityModule.register();
+```
+
+```ts
+import { IgcDataChartInteractivityModule } from 'igniteui-webcomponents-charts';
+
+IgcDataChartInteractivityModule.register();
+```
+
+## Data Chart UI Navigation Overview
```

Comment:
Change header to "Chart Navigation with User Interactions"

Reply:
pending